README.md

```diff
@@ -1,8 +1,63 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Omair Farooqui . Jarvis Consulting
 
+I am a Software Engineer who recently graduated from Ontario Tech University. I am someone who loves to learn and keep expanding my tool belt. I love this field because there is always constant room for growth and Software Engineers are in very high demand with all the tech in the world today. This field is a great place to be in as there as so many different roles and specializations you can consider. I am someone who has always found it easy to deal with the back-end of applications, however I would like to expand my role to Full Stack and learn a modern library like React to help achieve that.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Python, AWS, C++, C, JavaScript, HTML/CSS, Android, Docker, JSON
+
+**Familiar:** Flask, Google Cloud Platform, Blockchain, Heroku, UML, PHP
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/linux_sql)]: Designed an application to record the hardware specifications and monitor resources usages (CPU/memory etc) in real-time of each node within a Linux cluster of 10 nodes. These nodes will be running CentOS 7 and connected via switch, allowing them to communicate through internal IPv4 addresses. The data collected will be stored in a PostgreSQL database and used to generate reports for resource planning. The technologies used in this project include Bash (scripts), IntelliJ IDE, Docker, Git, and PostgreSQL.
```

Comment:
Designed an application to record the hardware specifications and monitor resources usages (CPU/memory etc) in real-time of each node within a Linux cluster. These nodes will be running CentOS 7 and connected via switch, allowing them to communicate through internal IPv4 addresses. The data collected will be stored in a PostgreSQL database and used to generate reports for resource planning. The technologies used in this project include Bash (scripts), IntelliJ IDE, Docker, Git, and PostgreSQL.

Developed XYX with A, B,C for purposes. Features. SQL, Docker.